### ISSUE TITLE
.tab('show') does not set the active class and hide deselected tabs

### ISSUE THREAD

**Issue by davidtgq** [twbs/bootstrap#21127]
http://codepen.io/anon/pen/GNZzoZ

The codepen demonstrates that:

1. Clicking the tabs behaves as expected, using data attributes.

2. Clicking the buttons only shows the panes; it does not set the `active` class on the nav-link or hide the panes of deselected tabs.

I realize that it's literally doing what "show" means, but if that argument is made, then this feature sacrifices usability for being technically correct. I cannot think of any scenario where a user would want to "show" a tab and not also hide the deselected panes or activate the corresponding tab in the nav. And if that scenario ever exists, then the user can simply use `$.show()` and `$.hide()`.

I propose that `$.tab('show')` or a new function such as `$.tab('select')` should automatically:
 - set the `active` class on the nav-link
 - hide the deselected panes

In other words, the function would emulate what happens when a tab is clicked, without having to use `.click()` (because that may have another event listener or popstate).

If this ends up in the WONTFIX pile, then the alternative is to update the [documentation](http://v4-alpha.getbootstrap.com/components/navs/#javascript-behavior) with an official/recommended method to manually hide the panes of deselected tabs.

**Comment by Johann-S** [comment]
The documentation say (http://v4-alpha.getbootstrap.com/components/navs/#tabshow) : 
> Selects the given tab and shows its associated pane.

So it's a normal behavior because, buttons in your CodePen test are not tabs

**Comment by hruship** [comment]
Hi @davidtgq 

I have found some issue with the reference. I have changed the reference to the tabs. It resolved the issue.

I have fixed the code and created a pen http://codepen.io/anon/pen/woWjab

Please refer to the same.

Regards

**Comment by davidtgq** [comment]
Thanks @hruship that resolves it for me, so I'd like to withdraw this issue, sorry and thanks for your time.

**Comment by bardiharborow** [comment]
@davidtgq if your issue is resolved, hit the "Close issue" button below.
